lib/internal/test_runner/mock/mock.js

```diff
@@ -61,9 +61,23 @@ class MockFunctionContext {
   mockImplementationOnce(implementation, onCall) {
     validateFunction(implementation, 'implementation');
     const nextCall = this.#calls.length;
-    const call = onCall ?? nextCall;
-    validateInteger(call, 'onCall', nextCall);
-    this.#mocks.set(call, implementation);
+    if (onCall) {
```

Comment:
```suggestion
    if (onCall !== undefined) {
```

Reply:
because `onCall` can be null I updated it to `onCall != undefined`

Reply:
Why would `onCall` be `null`, and why would we want to support that?

Reply:
it could have been null because of `??` and we don't want that to avoid another breaking change